src/components/application_manager/test/commands/mobile/on_system_request_notification_test.cc

```diff
@@ -100,12 +100,12 @@ TEST_F(OnSystemRequestNotificationTest, Run_ProprietaryType_SUCCESS) {
   EXPECT_CALL(mock_policy_handler, IsRequestTypeAllowed(_, _))
       .WillOnce(Return(true));
 
-#if defined(EXTENDED_POLICY) || defined(EXTERNAL_PROPRIETARY)
```

Comment:
@dev-gh Are you sure that this deletion is correct? I thought that `EXTERNAL_PROPRIETARY` flag was correct